driver/kubernetes/manifest/manifest.go

```diff
@@ -23,18 +29,25 @@ func NewDeployment(opt *DeploymentOpt) (*appsv1.Deployment, error) {
 	labels := map[string]string{
 		"app": opt.Name,
 	}
+	annotations := map[string]string{}
 	replicas := int32(opt.Replicas)
 	privileged := true
 	args := opt.BuildkitFlags
+
+	if len(opt.Platforms) > 0 {
+		annotations["platform"] = strings.Join(platformutil.Format(opt.Platforms), ",")
```

Comment:
Is this kubenetes standard or specific to buildx?

Reply:
specific for buildx, i want to read node platform.

k8s standard is the nodeSelector,  but not full information. so i use the custom annotation

Reply:
Should have buildx.docker.com/ prefix? 

cc @tonistiigi